src/index.js

```diff
@@ -22,6 +23,14 @@ export class AddonStore {
     this._preview = preview;
   }
 
+  getDatabase() {
+    return this._database;
+  }
+
+  setDatabase(database) {
```

Comment:
I really don't think we should restrict add-on developers from changing this. Developers may need to customize where and how they store their data.